Python/yaml/bigquery_to_kafka.yaml

```diff
@@ -0,0 +1,28 @@
+#  Copyright 2024 Google LLC
+
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+
+#       https://www.apache.org/licenses/LICENSE-2.0
+
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+pipeline:
+  type: chain
+
+  source:
+    type: ReadFromBigQuery
+    config:
+      table: project-id.dataset-id.table-id
+
```

Comment:
Could we add an example MapToFields and Filter to these? Doesn't need to be complicated, something similar to the csv_to_json filter is fine

Reply:
Done